packages/ui/components/proposal/ProposalYourVote.tsx

```diff
@@ -1,5 +1,27 @@
-export interface ProposalYourVoteProps {}
+import clsx from 'clsx'
 
-export const ProposalYourVote = ({}: ProposalYourVoteProps) => {
-  return <p>placeholder</p>
+export type YourVoteProps = {
+  variant: 'abstain' | 'pending' | 'no' | 'yes'
+}
+
+export const ProposalYourVote = ({ variant }: YourVoteProps) => {
+  return (
+    <div
+      className={clsx(
+        'flex relative justify-center items-center py-1 px-4 w-20 text-sm font-medium rounded-full',
+        variant === 'abstain'
+          ? 'text-text-secondary bg-primary'
+          : variant === 'pending'
+          ? 'border-2 border-inactive'
+          : variant === 'no'
+          ? 'text-text-interactive-error bg-[#C73E5914]'
+          : 'text-text-valid bg-[#39A69914]'
```

Comment:
```suggestion
          : 'text-text-valid bg-color-valid/[0.08]'
```

`#39A699` corresponds to `rgb(57, 166, 153)`, which is `color-valid`, so `bg-color-valid/[0.08]` should work.